src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_button_event_notification.cc

```diff
@@ -102,8 +102,10 @@ void OnButtonEventNotification::Run() {
       return;
     }
 
-    if ((mobile_api::HMILevel::HMI_FULL != app->hmi_level()) &&
-        (mobile_api::HMILevel::HMI_LIMITED != app->hmi_level())) {
+    const mobile_apis::HMILevel::eType app_hmi_level =
```

Comment:
Can buttons be in a widget outside of the default window? If yes, I believe this RPC needs to take into account the hmi level of the widget windows.

Reply:
@JackLivio Thank you for this finding. Defenetly button may belong to main or widget window. So appropriate window's HMI level needs to be taken into account.
We're preparing an update for core and scripts.

Reply:
@JackLivio fixed in 9862c6687b716c086c446c99ff0df04b5402bd2c